src/config/config.js

```diff
@@ -159,18 +165,18 @@ const config = convict({
     default: '63983fc2-cfff-45bb-8ec2-959e21062b9a'
   },
   azureClientSecret: {
-    doc: 'Azure App Client Secret',
+    doc: 'Azure App Client Secret. Defaults to stub secret',
     format: String,
     sensitive: true,
     env: 'AZURE_CLIENT_SECRET',
-    default: ''
+    default: 'test_value'
   },
   oidcWellKnownConfigurationUrl: {
-    doc: 'OIDC .well-known configuration URL',
+    doc: 'OIDC .well-known configuration URL. Defaults to the stub',
     format: String,
     env: 'OIDC_WELL_KNOWN_CONFIGURATION_URL',
     default:
-      'https://login.microsoftonline.com/6f504113-6b64-43f2-ade9-242e05780007/v2.0/.well-known/openid-configuration'
+      'http://cdp.127.0.0.1.sslip.io:3939/63983fc2-cfff-45bb-8ec2-959e21062b9a/v2.0/.well-known/openid-configuration'
```

Comment:
It feels like we are setting up this to work with stubs. Which feels a bit wrong/specific. Especially as stubs can set these values in stubs via environment variables. What happens if your not using stubs?

Reply:
You could make that case for any value you set in here right? Feels wrong we would add the real value of int-dev here. 

For service endpoints, these are configured to localhost etc. The config is usually configured to run locally where we override these values in app-config to run in the environments. This is consistent imo with how we set up local config. I don't think this is a smell - and it doesn't stop us from overriding locally via environments if that's what we want to do.